instagram/spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+feature "liking photos" do 
+    # {user = User.create}
+    # {user.Photo.create description: "Fujisan"}
+  
+  let (:user) {User.create(email: "test@test.com", 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.